js/app.js

```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
     bookFun.removeBook(targetEl);
   }
 });
+
+// Date and time
+const clockElement = document.getElementById('clock');
+
+function clock() {
+  clockElement.textContent = new Date().toString();
+}
```

Comment:
- [ ] Please according to javascript best practices you should use the arrow function instead of using the `function` keyword :smiley: